Dockerfile

```diff
@@ -1,18 +1,34 @@
-FROM node:15 as builder
-LABEL author="chevdor@gmail.com"
+FROM docker.io/library/node:15 as builder
 
 WORKDIR /opt/builder
 
 COPY . .
-RUN curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh -s -- -y && \
-    . $HOME/.cargo/env && \
-    cargo install wasm-pack && \
-    yarn install && \
+
+RUN yarn install && \
     yarn build
 
 # ---------------------------------
 
-FROM node:15-alpine
+FROM docker.io/library/node:15-alpine
+
+# metadata
+ARG VERSION=""
+ARG VCS_REF=master
+ARG BUILD_DATE=""
+
+LABEL summary="Substrate-api-sidecar." \
+	name="paritytech/substrate-api-sidecar" \
```

Comment:
Will this label need changing? Is it a reference to the repo or the docker namespace?

Reply:
oh thanks